typescript/index.d.ts

```diff
@@ -0,0 +1,160 @@
+// Type definitons for RDKit.js
+// Project: https://github.com/rdkit/rdkit-js
+// Definitions by: adam-of-barot <https://github.com/adam-of-barot>
+
+export interface JSMol {
```

Comment:
this was renamed from Mol to JSMol to match the naming in rdkit's minimallib files.